documentation/expanding.md

```diff
@@ -0,0 +1,196 @@
+
+    bolt plan run peadm::get_peadm_config --targets <primary-server-fqdn> 
+    bolt plan run peadm::add_replica primary_host=<primary-server-fqdn> replica_host=<new-replica-server-fqdn> replica_postgresql_host=<replica-postgres-server-fqdn>
+
+**Example**
+
+    % bolt task run peadm::get_peadm_config --targets pe-server-6251cd-0.us-west1-a.c.slice-cody.internal                                           -- INSERT --
+    Started on pe-server-6251cd-0.us-west1-a.c.slice-cody.internal...
+    Finished on pe-server-6251cd-0.us-west1-a.c.slice-cody.internal:
+      {
+        "params": {
+          "primary_host": "pe-server-6251cd-0.us-west1-a.c.slice-cody.internal",
+          "replica_host": null,
+          "primary_postgresql_host": "pe-psql-6251cd-0.us-west1-a.c.slice-cody.internal",
+          "replica_postgresql_host": "pe-psql-6251cd-1.us-west1-b.c.slice-cody.internal",
+          "compilers": [
+            "pe-compiler-6251cd-0.us-west1-a.c.slice-cody.internal",
+            "pe-compiler-6251cd-1.us-west1-b.c.slice-cody.internal"
+          ],
+          "compiler_pool_address": "puppet.pe-compiler-lb-6251cd.il4.us-west1.lb.slice-cody.internal",
+          "internal_compiler_a_pool_address": "pe-server-6251cd-0.us-west1-a.c.slice-cody.internal",
+          "internal_compiler_b_pool_address": null
+        },
+        "role-letter": {
+          "server": {
+            "A": "pe-server-6251cd-0.us-west1-a.c.slice-cody.internal",
+            "B": null
+          },
+          "postgresql": {
+            "A": "pe-psql-6251cd-0.us-west1-a.c.slice-cody.internal",
+            "B": "pe-psql-6251cd-1.us-west1-b.c.slice-cody.internal"
+          },
+          "compilers": {
+            "A": [
+              "pe-compiler-6251cd-0.us-west1-a.c.slice-cody.internal",
+              "pe-compiler-6251cd-1.us-west1-b.c.slice-cody.internal"
+            ],
+            "B": [
+    
+            ]
+          }
+        }
+      }
+    Successful on 1 target: pe-server-6251cd-0.us-west1-a.c.slice-cody.internal
+    Ran on 1 target in 2.56 sec
+
+## Adding Compilers with peadm::add_compiler
+
+Standard deployments are the only architecture of the three which do not have Compilers available upon initial deployment, the lack of them is what differentiates Standard from Large. Architecture has no effect on the process for adding Compilers to a deployment. The [peadm::add_compiler](https://github.com/puppetlabs/puppetlabs-peadm/blob/main/plans/add_compiler.pp) plan functions identical in all three architecture, even if you're adding the 1st or the 100th but some options do change slightly depending.
```

Comment:
* `Standard deployments are` -> `*Standard* deployment is`
* `do not` -> `does not`
* `three architecture` -> `three [deployment] architectures`
* `even if` -> `whether`
* something missing after last `depending`?